src/NzbDrone.Core/DecisionEngine/Specifications/MaximumSizeSpecification.cs

```diff
@@ -0,0 +1,47 @@
+﻿using NLog;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.IndexerSearch.Definitions;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.DecisionEngine.Specifications
+{
+    public class MaximumSizeSpecification : IDecisionEngineSpecification
+    {
+        private readonly IConfigService _configService;
+        private readonly Logger _logger;
+
+        public MaximumSizeSpecification(IConfigService configService, Logger logger)
+        {
+            _configService = configService;
+            _logger = logger;
+        }
+
+        public SpecificationPriority Priority => SpecificationPriority.Default;
+        public RejectionType Type => RejectionType.Permanent;
```

Comment:
What's the criteria for setting those two properties? I just guessed those values...

Reply:
So my guessing was correct... Thanks for the explanation